src/simulator/scenarios/US_runup.ts

```diff
@@ -0,0 +1,405 @@
+export const usHistory = [
+    {
+        days: -31,
```

Comment:
Note the negative days. This may look a bit odd, but the game engine uses a reference date as the start of the player's gameplay (Jan 1st 2021). Each game day, as calculated by the simulator increments the `days` property by one unit. Therefore negative indexes therefore represent dates in the past.